pynav/lib/states.py

```diff
@@ -64,8 +64,10 @@ def __init__(
     def plus(self, dx: np.ndarray):
         if self.direction == "right":
             self.value: np.ndarray = self.value @ self.group.Exp(dx)
+            return self.copy()
         elif self.direction == "left":
             self.value: np.ndarray = self.group.Exp(dx) @ self.value
+            return self.copy()
```

Comment:
i think its generally perferring to have a single return statement, for readability, when it is straightforward to do so